### PR TITLE
Fix hover on mobile and tablet devices for gl3d subplots

### LINKED ISSUE #4550: Gl3d hover not working on mobile and tablet since v1.34.0
Since [v1.34.0](https://github.com/plotly/plotly.js/releases/tag/v1.34.0) the gl3d hover not working on mobile and tablet devices 
[working demo using 1.33.x](https://codepen.io/MojtabaSamimi/pen/ZEGzrXJ)

### PR DESCRIPTION
**Pull request by jdpaterson**
Update gl-plot3d version to fix #4550 .
Related to https://github.com/gl-vis/gl-plot3d/pull/31

### CODE REVIEW

**Comment by archmoj** [comment]
Thanks very much for the PR!
There is only one test failing now.
https://github.com/plotly/plotly.js/blob/0fce11be72f4b6e630ba85e3c0a0b9023f8d7e6d/test/jasmine/tests/gl3d_plot_interact_test.js#L929-L971
@jdpaterson could you please investigate why?

File: test/jasmine/tests/gl3d_plot_interact_test.js [hunk 1]
```diff
@@ -941,15 +941,15 @@ describe('Test gl3d drag and wheel interactions', function() {
         var sceneTarget;
         var relayoutCallback = jasmine.createSpy('relayoutCallback');
 
-        function assertEvent(e) {
-            expect(e.defaultPrevented).toEqual(true);
+        function assertEvent(e, passive) {
+            expect(e.defaultPrevented).toEqual(!passive);
```
Comment (archmoj) [inline test/jasmine/tests/gl3d_plot_interact_test.js:945]: Non-blocking: 
```js
expect(e.defaultPrevented).not.toEqual(passive);
```

**Comment by jdpaterson** [comment]
@archmoj the test-bundle timed out during the mapbox plot tests. Is this something I should be concerned about here?

**Comment by archmoj** [comment]
> @archmoj the test-bundle timed out during the mapbox plot tests. Is this something I should be concerned about here?

@jdpaterson No. I'll re-run the container tomorrow.
Also I'll spend more time testing.
Thanks again for the fantastic work.

File: package.json [hunk 2]
```diff
@@ -86,7 +86,7 @@
     "gl-mat4": "^1.2.0",
     "gl-mesh3d": "^2.3.1",
     "gl-plot2d": "^1.4.5",
-    "gl-plot3d": "^2.4.6",
+    "gl-plot3d": "git://github.com/jdpaterson/gl-plot3d.git#8f96f0362366394a188b2f7451faf4b48da85433",
```
Comment (archmoj) [inline package.json:89]: Please bump `gl-plot3d` version to "^2.4.7" here. Save and `npm install`. 
Then commit `package.json` and `package-lock.json`.
Thanks!

Comment (jdpaterson) [reply package.json]: Done. Thanks for your help @archmoj !

**Comment by archmoj** [comment]
Nicely done! :1st_place_medal: :medal_military: @jdpaterson 
:dancer:

**Pull request merged**